manager/src/main/java/com/redhat/service/smartevents/manager/BridgesServiceImpl.java

```diff
@@ -135,12 +182,10 @@ public void deleteBridge(String id, String customerId) {
         }
 
         Bridge bridge = findByIdAndCustomerId(id, customerId);
-        if (!isBridgeDeletable(bridge)) {
+        if (!isBridgeActionable(bridge)) {
             throw new BridgeLifecycleException("Bridge could only be deleted if its in READY/FAILED state.");
         }
 
-        hiddenProcessors.getItems().forEach(p -> processorService.deleteProcessor(id, p.getId(), customerId));
```

Comment:
I have moved deletion of the Error Handler processor to `BridgeWorker` for consistency.